aspnetcore/signalr/scale.md

```diff
@@ -104,14 +104,70 @@ The preceding conditions make it likely to hit the 10 connection limit on a clie
 
 ## Linux with Nginx
 
-Set the proxy's `Connection` and `Upgrade` headers to the following for SignalR WebSockets:
+The following contains the minimum required settings to enable WebSockets, ServerSentEvents, and LongPolling for SignalR:
 
 ```nginx
-proxy_set_header Upgrade $http_upgrade;
-proxy_set_header Connection $connection_upgrade;
+http {
+  map $http_upgrade $connection_upgrade {
+    default Upgrade;
+    '' close;
+  }
+
+  server {
+    listen 80;
+    server_name example.com *.example.com;
+
+    root /path/to/wwwroot;
+
+    # Configure the SignalR Endpoint
+    location /hubroute {
+      # App server url
+      proxy_pass http://localhost:5000;
+
+      # Configuration for WebSockets
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection $connection_upgrade;
+      proxy_cache_bypass $http_upgrade;
```

Comment:
If we worried about the proxy caching requests, shouldn't we be worried about every request, not just WebSocket requests?

```suggestion
      proxy_cache off;
```